addons/account/models/account_move.py

```diff
@@ -3342,13 +3342,13 @@ def _reverse_moves(self, default_values_list=None, cancel=False):
                 skip_invoice_sync=move.move_type == 'entry',
             ).copy(default_values)
 
-        reverse_moves.with_context(skip_invoice_sync=cancel).write({'line_ids': [
```

Comment:
Not sure why this was needed, locally it doesn't seems to fail any test

Reply:
no need for the diff it if it's not related to our fix

Reply:
it's related, we need the invoice sync to recalculate epd discount on tax and adjust the payment terms, but maybe there's another way to have the same results